internal/pkg/service/buffer/model/model.go

```diff
@@ -0,0 +1,215 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type ImportCondition struct {
+	Count int               `json:"count" validate:"min=1,max=10000000"`
+	Size  datasize.ByteSize `json:"size" validate:"min=100,max=50000000"`
+	Time  time.Duration     `json:"time" validate:"min=30s,max=24h"`
+}
+
+type Export struct {
+	ID               string            `json:"exportId" validate:"required,min=1,max=48"`
+	Name             string            `json:"name" validate:"required,min=1,max=40"`
+	ImportConditions []ImportCondition `json:"importConditions" validate:"required"`
+}
+
+type TableID struct {
+	Stage      string `json:"stage"`
+	BucketName string `json:"bucketName"`
+	TableName  string `json:"tableName"`
+}
+
+func (t TableID) String() string {
+	return fmt.Sprintf("%s.c-%s.%s", t.Stage, t.BucketName, t.TableName)
+}
+
+type Mapping struct {
+	RevisionID  int     `json:"revisionId" validate:"required"`
+	TableID     TableID `json:"tableId" validate:"required,min=1,max=198"`
```

Comment:
Je to struktura, nemoze sa validovat `min/max`.
Je pekne ze to je podelene na samostatne fields, validacia teda pojde do `TableID` struktury,
tu su hodnoty: https://keboola.slack.com/archives/CFVRE56UA/p1666709705352709?thread_ts=1666708471.297969&cid=CFVRE56UA

Reply:
(neviem kto to pisal, vsimol som si to az teraz, nemusi sa to riesit v tomto PR)